Packaging deprecation and installation updates

ferki opened the pull request:
This is a PR towards fixing both #104 and #116. These two doesn't really
makes sense without each other, so let's collect the related changes
here and then merge them together.

I'll remove the draft status when I consider it to be ready, but I'm
still sharing it early so it's possible to take a look ahead of time.

ferki commented:
@ehuelsmann @krimdomu: I think I'm ready with preparing the related changes, so I'd like to get your feedback on it.

One think I noticed, is CentOS and SuSE doesn't seem to have Rex packages in their own main repos.

@krimdomu: would you like to keep building packages for those and sending them to their upstream? If yes, I can reword the "Get Rex" page to list those as community repos.

ferki commented:
Hmm, I'm considering to replace the list of operating systems with packages to simply point to repology: https://repology.org/project/rex/versions

That would be automatically kept up-to-date, and probably would always be a more complete list than what we would compile on our own manually.

ferki commented:
I've reworded the instructions to:
 - point to the Repology page of Rex for available packages
 - have a placeholder for community overlays/repos

The CentOS and (open)SuSE packages on rex.linux-files.org are old/unmaintained (latest package is rex-1.4.1 from 2016-07-16), and/or built for an OS version that is EOL since then (except for CentOS 7, but then again that is the now 3 years old rex-1.4.1). In any case, it would be easy to point to them later, if needed.

@ehuelsmann @krimdomu: please review the content, and mark the PR either approced or request changes. I'll handle merging and deployment (=I'd like to update the timestamp of the announcement right before merging).

ehuelsmann commented on blog/2019/06/26/deprecating_official_rex_packages.md:
```diff
@@ -0,0 +1,35 @@
+
+Many operating systems are already packaging and shipping Rex in their own respective repositories, so building packages on our own feels like a duplicate effort.
+
+We believe the OS-specific packages in general are also having much better quality than we could achieve on our own. They are maintained by their upstreams who are in much better position to follow changes of the packaging guidelines for the given distribution.
```
`in general are also having much` -> `in general also have much` ?

ferki replied:
reworded as `in general are also much better in quality`

ehuelsmann commented on blog/2019/06/26/deprecating_official_rex_packages.md:
```diff
@@ -0,0 +1,35 @@
+
+We felt this is too far outside the scope of the Rex project, and we wished we could spend the same amount of effort on Rex itself instead.
+
+## Releases are much simpler with CPAN
```
maybe refer to `local::lib` for the 'not a super-user' installations?

ferki replied:
I've slightly rewritten that part to mention the local user's own perl environment, and noted local::lib, perlbrew, etc, by linking to them. Plus added a remark about system wide installation pointing to packages.

ehuelsmann commented on content/get/index.md:
```diff
@@ -1,265 +1,32 @@
+### Official packages
 
-Tested with Windows Vista and Windows 7 (64bit) and Strawberry Perl 5.x
+Rex is distributed via package managers for many different operating systems. The [Repology page of Rex](https://repology.org/project/rex/versions) is perhaps the most complete and up-to-date list of available Rex packages.
```
Nice find! I think it's a really good idea to depend on an external source like that!

ferki replied:
Yes, I rediscovered the site just in time for this PR :) I'm glad you both like this approach!

ferki commented:
Thanks @krimdomu and @ehuelsmann for the feedback! :+1: I'll refine the wording of the bits mentioned above, put a fresh timestamp on it, and then merge.

ferki commented on blog/2019/06/26/deprecating_official_rex_packages.md:
```diff
@@ -0,0 +1,35 @@
+
+---
+
+There are multiple reasons behind this decision, and I'd like to mention a few bigger one below.
```
reworded into `some of the bigger ones`

ferki commented on blog/2019/06/26/deprecating_official_rex_packages.md:
```diff
@@ -0,0 +1,35 @@
+
+## Existing upstream packages in various distributions
+
+Many operating systems are already packaging and shipping Rex in their own respective repositories, so building packages on our own feels like a duplicate effort.
```
reworded as `building our own packages`

ferki commented:
@krimdomu @ehuelsmann : many thanks for the feedback, I'm glad you both agree with the content! I've addressed all the remarks, updated the timestamp, and will deploy the announcement soon.
